# hermes_ip: "192.168.43.1"
hermes_ip: "192.168.1.5"

linux_ip: "192.168.1.7"
# linux_ip: "192.168.43.223"

subscribe_asset_port: 10000
start_asset_port: 10001
stop_asset_port: 10002
asset_state_port: 10003
get_identity_port: 10004
send_signal_port: 10005
port_pool_range: [10010, 10500]

# http://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/Imu.html
imu:
  - name: "imu0"
    select:
      id: "in72e"
      fps: 30
    covariance:
      orientation_covariance_diagonal: [0.05, 0.004, 0.003]
      angular_velocity_covariance_diagonal: [0.05, 0.004, 0.003]
      linear_acceleration_covariance_diagonal: [0.05, 0.004, 0.003]

# http://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/NavSatFix.html
# http://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/NavSatStatus.html
gnss:
  - name: "gnss"
    select:
      id: "A4e2r"
      fps: 10 
    
# http://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/CameraInfo.html
# http://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/CompressedImage.html
camera:
  - name: "arducam0"
    frame_id: "arducam0"
    select:
      id: "2"
      stream:
        fps: 30
        width: 1280
        height: 720
        pixel_format: "mjpeg"
      compression_quality: 80
    camera_info_url: "file://home/ubuntu/.ros/flo/arducam0.yaml"
  - name: "arducam1"
    frame_id: "arducam1"
    select:
      id: "3"
      stream:
        fps: 30
        width: 1280
        height: 720
        pixel_format: "mjpeg"
      compression_quality: 80
    camera_info_url: "file://home/ubuntu/.ros/flo/arducam1.yaml"

# http://docs.ros.org/en/melodic/api/std_msgs/html/msg/String.html
speaker:
  - name: "speaker"
    select:
      id: "0"
      language: "en"

usb_serial:
  - name: "arduino0"
    select:
      id: "0"
      baud: 115200
      delimiter: "\n"
  - name: "arduino1"
    select:
      id: "1"
      baud: 115200
      delimiter: "\n"
